### ISSUE TITLE
Create Homepage with twitter etc.

### ISSUE THREAD

**Issue by bocian67** [Dezentrale-eV/dezentrale_web#26]
The home/landingpage should provide at least the following components:
- [x] twitter banner
- [ ] basic self-publishing of dezentrale (in progress so far)

**Comment by elbenfreund** [comment]
Maybe it would be good to split this issue? Such as "create basic homepage". And then "Add XYZ" as separate issues? Also, isn't this depending on #16? Does it make sense without the base layout finished?

**Comment by bocian67** [comment]
Yeah, the template should be finished, but I cannot contact the guy who has it, now I just create all contents and then just merge the template and our work. But yes it would be ideal to first have a template and then the contents.

**Comment by bocian67** [comment]
@elbenfreund is it correct how I included the migrations?

**Comment by bocian67** [comment]
@elbenfreund pls help out ;) How to include which migration?

**Comment by elbenfreund** [comment]
I will reply in the relevant PR :) I guess it is #32 ? Please not that this PR seems to have a totaly different scope than *this* issue. At least according to the title.

**Comment by elbenfreund** [comment]
Neverrmind my last comment. PR #32 **is*** something else entirely :)
I guess you are refering to ``f672f2ab8de878aa31ad71aafd4ceeb14b0559d0``?

**Comment by elbenfreund** [comment]
I have cleaned up the commits and rebased the branch on ``feature/55``. As you will notice, I moved the changes to the ``blog`` templates to a separate issue (#55). It is important that PRs have the smallest scope possibe and having changes to multiple apps in one PR is always a good indication things went wrong... :)

**Comment by elbenfreund** [comment]
Once #56 is merged I think we should open a PR for ``feature/#26`` and discuss you changes 👍
